packages/teams/src/utils/get-user-creatable-teams.ts

```diff
@@ -4,27 +4,38 @@ import { WELLKNOWNTEAMS } from "../well-known-teams";
 import { PRE_9_1_WELLKNOWNTEAMS } from "../pre-9-1-well-known-teams";
 import { canUserCreateTeamInProduct } from "./can-user-create-team-in-product";
 import { IGroupTemplate } from "../types";
+import { filterUserPrivsBySubscriptionType } from "./filter-user-privs-by-subscription-type";
+import { updateTemplateBasedOnPrivs } from "./update-template-based-on-privs";
 
 /**
  * Return array of group templates that the current user has licensing
  * and privs to create in the current environment (AGO vs Portal)
  * @param {object} user
  * @param {string} environment
+ * @param {string} portalApiVersion
+ * @param {string} subscriptionInfoType
  */
 export function getUserCreatableTeams(
   user: IUser,
   environment: HubProduct,
-  portalApiVersion: string
+  portalApiVersion: string,
+  subscriptionInfoType: string
```

Comment:
tbh - not super wild about passing in this deeply nested property value. makes testing easy, but seems like it's "leaking" an abstraction. Since it's currently a breaking change (i.e. we're adding an argument, that does not have a default value, ergo clients that don't pass in a value will break)... either, give this a default value (so it's a minor release) or just ask for an IPortal, vs the version + subscription info.

Reply:
As discussed going with the default empty string value.